pom.xml

```diff
@@ -1792,6 +1792,11 @@
             <version>1.8.1</version>
         </dependency>
  		<!-- //BiDiB -->
+      <dependency>
+        <groupId>org.reflections</groupId>
+        <artifactId>reflections</artifactId>
+        <version>0.10.2</version>
+      </dependency>
```

Comment:
`javassist-3.28.0-GA.jar` is included in `build.xml` but not in `pom.xml`, unless I'm missing something.

Reply:
javassist has never been referenced directly in the pom.xml, even though it has been used for while.  Maven doesn't require explicit references to dependencies of dependencies.